versioned_docs/version-latest/faq.md

```diff
@@ -6,7 +6,13 @@ This FAQ is a work in progress designed to answer the questions our users most f
 
 #### **Q: Is Rancher Desktop a desktop version of Rancher?**
 
-**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop allows users to locally run a container management platform optionally including Kubernetes. The two solutions complement each other.
+**A:** No. Rancher Desktop is not Rancher on the Desktop. 
+
+While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. But, the two solutions do complement each other:
+
+Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure.
+
+Rancher Desktop provides container management and an optional Kubernetes instance on the desktop to support local development use cases.
```

Comment:
The use of `optional`, even though technically correct, deemphasizes the fact that Rancher Desktop takes a Kubernetes first approach, and sounds like Kubernetes is available as just an option. Can you please remove the word `optional`?

Reply:
sold